scripts/action-recognition/test_recognizer.py

```diff
@@ -157,7 +190,7 @@ def test(ctx, val_data, opt, net):
             pred = net(X.astype(opt.dtype, copy=False))
             if opt.use_softmax:
                 pred = F.softmax(pred, axis=1)
-            pred = F.mean(pred, axis=0, keepdims=True)
+            # pred = F.mean(pred, axis=0, keepdims=True)
```

Comment:
@wuxun-zhang I changed the dataloader for video action recognition a month ago, it might only support `batch_size=1` during testing at this moment, but I'm not sure. Let me look into this.

Reply:
I just commented out this, and it works well when batch-size>1.

Reply:
The reason it works when `batch-size>1` if you comment out that line, is because the default `num_segments=1` and default data augmentation during test is `single center crop`. But if we have more segments or using ten-crop, it will fail too. 

For example, if you use default setting, the label is 64x1 vector, and the prediction is also 64x1 vector, so it doesn't raise any error. However, if you use three segments and ten-crop, then the label is still 64x1, but the prediction now is 1920x1. The shape does not match. 

The test code and dataloader right now only support `batch_size=1`. Since this is not high priority, I didn't look into this for a long time. If you need `batch_size>1`, I can go ahead fix it and do a PR. This will take a day or two.

Reply:
I see. Thanks for your explanation. 

Just running test mode on CPU, `batch_size>1` will bring some speedup over `batch_size=1`. If it expects not too much effort, please help fix this to support `batch_size>1`. Of course, it is not urgent since it is not the dependency of this PR. I will add this line back.